frontend/src/lib/forms/MigrationStatusSelect.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+  import { ProjectMigrationStatus } from '$lib/gql/generated/graphql';
+  import Select from './Select.svelte';
+
+  export let value: ProjectMigrationStatus | 'UNMIGRATED' | undefined;
+  export let error: string | string[] | undefined = undefined;
+
+</script>
+<div class="relative">
+  <Select id="migration-status" label="Migration status" bind:value {error} on:change>
+    <!-- No need to translate these as this is admin-only -->
```

Comment:
The main reason I didn't translate actually is not because it's admin only but because we're going to remove all this code once migration is finished.

Reply:
Will skip translation as suggested by Kevin.